ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.hpp

```diff
@@ -49,18 +45,13 @@ struct Conv2dConfig {
     bool enable_split_reader = false;
     bool enable_subblock_padding = false;
     static constexpr auto attribute_names = std::make_tuple(
```

Comment:
i think the `attribute_names` is no longer needed, we can delete.

Reply:
What about `attribute_values`?

Reply:
that as well